chrome/content/zotero/xpcom/utilities.js

```diff
@@ -1527,14 +1484,14 @@ Zotero.Utilities = {
 	 */
 	"itemToCSLJSON":function(zoteroItem) {
 		if (zoteroItem instanceof Zotero.Item) {
-			zoteroItem = zoteroItem.toArray();
+			zoteroItem = Zotero.Utilities.Internal.itemToExportFormat(zoteroItem);
```

Comment:
For now I'll continue mapping these to `article`, because there are plenty of styles that do reference `article` explicitly. On the other hand, I doubt that any of these styles are actually attempting to format `document` items or file attachments/notes.